pydarn/plotting/fan.py

```diff
@@ -0,0 +1,243 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Daniel Billett, Marina Schmidt
+"""
+Fan plots, mapped to AACGM coordinates
+"""
+
+import datetime as dt
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+
+from matplotlib import ticker, cm, colors
+from typing import List
+
+# third part libs
+import aacgmv2
+
+from pydarn import PyDARNColormaps
+
+class Fan():
+    # TODO: Add class documentation
+
+    def __str__(self):
+        return "This class is static class that provides"\
+                " the following methods: \n"\
+                "   - plot_radar_fov()\n"\
+                "   - return_beam_pos()\n"
+
+    @classmethod
+    def plot_radar_fov(cls, dmap_data: List[dict], scan_index: int = 0,
```

Comment:
```suggestion
    def plot_radar_fov(cls, dmap_data: List[dict], ax=None, scan_index: int = 0,
```

Not sure how to do type suggestions with NoneType/axes class input.